skipper.go

```diff
@@ -1728,6 +1732,7 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 			builtin.NewRouteCreationMetrics(mtr),
 			fadein.NewPostProcessor(),
 			admissionControlSpec.PostProcessor(),
+			failClosedRatelimitPostProcessor,
```

Comment:
I only add the postprocessor in case we have ratelimits, because otherwise it does not make sense to add.
I now check for nil before adding it to the PostProcessors